Database/Procedures/2020_12_16_002_SP_UpdateHotel.sql

```diff
@@ -0,0 +1,38 @@
+USE [Hotels_DB]
+GO
+/****** Object:  StoredProcedure [dbo].[UpdateHotel]    Script Date: 16.12.2020 17:38:10 ******/
+SET ANSI_NULLS ON
+GO
+SET QUOTED_IDENTIFIER ON
+GO
+-- =============================================
+-- Author:		<Author,,Name>
+-- Create date: <Create Date,,>
+-- Description:	<Description,,>
+-- =============================================
+CREATE PROCEDURE [dbo].[UpdateHotel]
+@HotelID int,
+@Country int,
+@Owner int,
+@Number_of_stars int,
```

Comment:
Write variable in _CamelCase_. I guess it will be **better**

Reply:
I wrote the variables as they are written in the table. So can't say for sure, if I should write in CamelCase

Reply:
17 line, your own variable

Reply:
![IMG_20201217_110541_290.jpg](https://user-images.githubusercontent.com/50269023/102466572-daac7600-4057-11eb-8a07-0e537ec643f6.jpg)
That's why I called it like this

Reply:
okey, as you want

Reply:
This name of variable? which is used in table, is being used in sql tables, so I also think that, when you write variables with @, it will be better to use CamelCase.